rust/uefi/linux-bootloader/src/linux_loader.rs

```diff
@@ -179,6 +154,21 @@ impl InitrdLoader {
     }
 }
 
+/// Builds the device path for the LINUX_EFI_INITRD_MEDIA protocol.
+/// The protocol is associated with the [`LINUX_EFI_INITRD_MEDIA_GUID`].
```

Comment:
Can you leave the link to the u-boot commit? That's pretty helpful, if people want to understand what's going on here.